frontend/tickets-online/src/sessions/components/Session.js

```diff
@@ -9,21 +10,28 @@ class Session extends React.Component {
     super(props);
   }
 
+  onClick = e => {
+    e.preventDefault();
+    this.props.onSessionClick(this.props.hallId, this.props.id, this.props.cinemaName);
+  };
+
   render() {
     return (
       <a
         href="#"
         className="film-item list-group-item list-group-item-action flex-column align-items-start"
+        onClick={this.onClick}
       >
         <div className="d-flex w-100 justify-content-between">
           <h5 className="mb-1">
-            <strong>{this.props.filmName}</strong>
-            {this.props.sessionDate}
+            <strong>{this.props.filmName} </strong>
+            <br />
+            {moment(this.props.sessionDate).format('MMMM Do YYYY, hh:mm:ss a')}
```

Comment:
Advice for future: extract such formats to some settings to reuse between components and have possibility quickly change them.